plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -177,6 +177,34 @@ func PodMatchesNodeLabels(pod *api.Pod, node *api.Node) bool {
 	return selector.Matches(labels.Set(node.Labels))
 }
 
+// NoPodLabelConflict evaluates if a pod fits a node by:
+// 1) Comparing the pod's conflict labels against labels of all the other pods.
+// 2) Comparing all the other pods conflict labels with labels of this pod.
+func NoPodLabelConflict(pod *api.Pod, existingPods []*api.Pod, node string) (bool, error) {
+	for podIx := range existingPods {
+		if existingPods[podIx].Labels != nil {
+			for conflict, value := range pod.Spec.Conflicts {
+				if existingValue, ok := existingPods[podIx].Labels[conflict]; ok {
```

Comment:
If we were going to add conflicts, we should use standard label selection semantics, as I described here:
https://github.com/GoogleCloudPlatform/kubernetes/issues/4301#issuecomment-74355529